src/app/pages/RewardPage/components/RewardsDetail/index.tsx

```diff
@@ -84,11 +84,16 @@ export const RewardsDetail: React.FC<IRewardsDetailProps> = ({
           <LoadableValue
             value={
               bignumber(availableAmount).greaterThan(0) ? (
-                <Tooltip content={`${weiTo18(availableAmount)} ${asset}`}>
+                <Tooltip content={`${availableAmount} ${asset}`}>
                   <>
-                    {showApproximateSign && '≈ '}
-                    {weiToNumberFormat(availableAmount, 6)}
-                    <span className="tw-mr-1">...</span> {asset}
+                    <AssetValue
+                      value={Number(availableAmount)}
```

Comment:
Due to the `weiToXX()` formatting calls being omitted here now, the fees totals shown on Fees Earned tab are broken - please update `src\app\pages\RewardPage\components\FeesEarnedTab\index.tsx` with the following change (addition of `weiTo18()`'s):

```
 <RewardsDetail
          color={RewardsDetailColor.Yellow}
          title={t(translations.rewardPage.fee.stakingFee)}
          availableAmount={weiTo18(amountToClaim)}
          totalEarnedAmount={weiTo18(totalRewardsEarned)}
          asset={Asset.RBTC}
          loading={totalAmountLoading}
          showApproximateSign
        />
```